### DISCUSSION TITLE
Does Github pages support CGI Scripts?

### DISCUSSION THREAD

**Discussion by rdcr0** [community/community#23797] [Other Feature Feedback, Questions, & Ideas]
<p>Does Github Pages support CGI Scripts for a website?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Hello <a class="mention" href="/whoami1000">@WhoAmI1000</a>, thanks for joining the community.</p>
<p>No, GitHub Pages does not offer any sort of server-side code execution, including CGI scripts.</p>
